README.md

```diff
@@ -22,6 +22,20 @@ username: admin
 password: admin
 ```
 
+## Production Deploy
+
+- Run `./build-prod.sh` on machine with access to our Quay registry
+- Run `./provsn jenkins run './start-prod.sh $VERSION'` from ipfs/infrastructure
```

Comment:
provsn doesn't have a jenkins command, do you mean something like `./provsn exec jenkins '/path/to/start-prod.sh $VERSION'?